akka-actor-tests/src/test/scala/akka/actor/FSMTransitionSpec.scala

```diff
@@ -51,7 +51,8 @@ object FSMTransitionSpec {
     }
     onTransition {
       case 0 -> 1 ⇒ target ! ((stateData, nextStateData))
-      case 1 -> 1 ⇒ target ! ((stateData, nextStateData))
+      // Verify the unicode alias of -> works correctly in onTransition
+      case 1 → 1  ⇒ target ! ((stateData, nextStateData))
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Non ascii characters are not allowed](https://www.codacy.com/app/vasseurjb/akka/file/2046658181/issues/source?bid=3132306&fileBranchId=3132374#l55)